src/views/portfolio/projects/Project.vue

```diff
@@ -247,11 +248,17 @@
     },
     mounted() {
       this.initialize();
+      if (this.$route.params.componentUuid){
+        this.$refs.tabDependencyGraph.active = true
+      }
     },
     watch:{
       $route (to, from){
         this.uuid = this.$route.params.uuid;
         this.initialize();
+        if (this.$route.params.componentUuid){
+          this.tabIndex = 3
```

Comment:
Can we select this by a reference or name? using the index might break in future changes, also, depending on user permissions some tabs might not be visible for the current user.